docs/settingup.md

```diff
@@ -183,6 +183,9 @@ This section of the JSON file contains scheduler settings for the users in the l
   * `reuseUsers`: 
       * `true`: Every iteration for each concurrent user uses the same user and session.
       * `false`: Every iteration for each concurrent user uses a new user and session. The total number of users is the product of `concurrentusers` and `iterations`.
+  * `onlyinstanceseed`: Disable session part of randomization seed. Defaults to `false` if omitted.
+      * `true`: All users and sessions will have the same randomization sequence, only changing if `instance` flag is changed.
```

Comment:
Slight rewrite suggested:
"All users and sessions have the same randomization sequence, which only changes if the 'instance' flag is changed."